include/authorization.hpp

```diff
@@ -240,6 +240,16 @@ static bool isOnWhitelist(std::string_view url, boost::beast::http::verb method)
         }
     }
 
+    // This patch is allowed for service user, without authorization to upload
+    // unauthenticated ACF.
+    if (boost::beast::http::verb::patch == method)
+    {
+        if (url == "/redfish/v1/AccountService/Accounts/service")
```

Comment:
I don't think we want this.. 
This will allow unauthorized user to PATCH service, including enabling/disabling service

Reply:
Why can't we handle the PATCH here via normal auth?